src/components/FormSignIn.vue

```diff
@@ -72,14 +77,19 @@ import { asset, api, url } from '@lib/helpers';
 import { SITE_TITLE_SHORT } from '@lib/constants';
 import { ref } from 'vue';
 
+const form = ref<HTMLFormElement>();
 const formFields = ref({
 	email: 'test@test.com',
 	password: 'test@test.com',
```

Comment:
Let's take this opportunity to remove the default values.
```suggestion
	email: '',
	password: '',
```